1-js/02-first-steps/04-variables/article.md

```diff
@@ -24,7 +24,8 @@ let message;
 let message;
 
 *!*
-message = 'سلام'; // ذخیره کن message رشته «سلام» را در متغیر
+ //ذخیره کن message رشته «سلام» را در متغیر
+message = 'سلام';
```

Comment:
This will cause merge conflicts in the future upstream PRs.

Maybe it's better to translate the string to "Hello", so the comment wouldn't be combined with it.